src/Microsoft.Health.Fhir.SqlServer.Api.UnitTests/Controllers/SchemaControllerTests.cs

```diff
@@ -58,11 +61,5 @@ public void GivenACurrentVersiontRequest_WhenNotImplemented_ThenNotImplementedSh
         {
             Assert.Throws<NotImplementedException>(() => _schemaController.CurrentVersion());
         }
-
-        [Fact]
-        public void GivenACompatibilityRequest_WhenNotImplemented_ThenNotImplementedShouldBeThrown()
```

Comment:
Since we're removing the error test, do we want to add a success test?

Reply:
Test is added in the  LatestVersionSchemaHandler for success scenario  and there is no particular check in controller before it gets processed by the handler. Let me know if we still need one in controller